deployer/src/deployment/provisioner_factory.rs

```diff
@@ -101,7 +101,7 @@ impl<R: ResourceRecorder, S: SecretGetter> Factory for ProvisionerFactory<R, S>
         let db_type: DbType = db_type.into();
 
         let request = Request::new(DatabaseRequest {
-            project_name: self.project_name.to_string(),
+            project_name: self.service_name.to_string(),
```

Comment:
Renaming this field means we will also need to rename it in the protobuffer and `provisioner`, not sure if that is what we want.

Reply:
I agree with you, we shouldn't rename this.